libcloudforensics/providers/aws/internal/ec2.py

```diff
@@ -361,6 +360,9 @@ def GetOrCreateAnalysisVm(
       tags (Dict[str, str]): Optional. A dictionary of tags to add to the
           instance, for example {'TicketID': 'xxx'}. An entry for the instance
           name is added by default.
+      subnet_id (str): Optional. Subnet to launch the instance in
+      userdata (str): Optional. String passed to the instance as a userdata
```

Comment:
Maybe change this to 
```
    userdata_file (str): Optional. Filename to be read in as the userdata launch script.
```
So it's consistent with `aws.forensics`? + `.` at the end of the sentences.

Reply:
It made more sense to me to read the userdata file in `forensics.py` and have the userdata string passed in to the vm creation method - in the context of a future feature addition which also will launch an instance, that isn't a forensics analysis vm.

Reply:
Makes sense, thanks!